### PR TITLE
Update Video Detect region tags.

### PR DESCRIPTION
**Pull request by kurtisvg**

### CODE REVIEW

File: videointelligence/video_analyze/video_analyze.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 	"golang.org/x/net/context"
 )
 
+// [START video_analyze_labels_local]
```
Comment (tbpg) [inline videointelligence/video_analyze/video_analyze.go]: Would you mind adding a newline between the region tag comments and the function declaration? As is, the region tag comments will end up in the function GoDoc.

Comment (kurtisvg) [reply videointelligence/video_analyze/video_analyze.go]: How does that look?

Comment (tbpg) [reply videointelligence/video_analyze/video_analyze.go]: LGTM. I ran `gofmt` since it was complaining.

**Comment by tbpg** [comment]
Holding off on merging for now.

**Pull request merged**